### DISCUSSION TITLE
fzf.fish keybindings no longer work with update

### DISCUSSION THREAD

**Discussion by dessalines** [PatrickF1/fzf.fish#168] [Q&A] [answered]
**Describe the bug**

I had some problems with fzf.fish, so I tried removing it with fisher, and reinstalling. Now no keybindings are in `fish_variables`.

**To Reproduce**

- Uninstall fzf.fish with fisher, reinstall. Doesn't work, keybindings missing.

**Expected behavior**
It to work.

**Screenshots**

**Environment**
- Fish: 3.1.2
- fzf.fish: v5.5
- plugin manager: fisher 4.2.0
- terminal: alacritty
- OS: arch linux

**Additional context**

Is there any way to test that it works correctly, even without keybindings?

**Comment by dessalines** [comment, marked as answer]
Nevermind, fisher has a weird behavior on my system where it won't install the function files unless I start a fresh term.

**Comment by jorgebucaran** [comment]
Feel free to open an issue in the Fisher repository so we can look into it. It could be something in your `config.fish`.

**Comment by PatrickF1** [comment]
Glad you figured it out!
